### DISCUSSION TITLE
Could you make most used languages' width modifiable ?

### DISCUSSION THREAD

**Discussion by YChienHung** [anuraghazra/github-readme-stats#2170] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

look, it is unbeautiful.
If the width can be modified, it can make the picture more beautiful.
![image](https://user-images.githubusercontent.com/41673864/194745463-d160e7dd-4dcf-4491-a7ed-107b0786f5e4.png)

**Comment by rickstaa** [comment, marked as answer]
@YChienHung Thanks for your question request. We offer support for this through the `card_width` argument.

**card_width=250**

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&card_width=250)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&card_width=250)](https://github.com/anuraghazra/github-readme-stats)

**card_width=500**

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&card_width=500)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&card_width=500)](https://github.com/anuraghazra/github-readme-stats)

**Reply by rickstaa** [reply]
@YChienHung  Does that answer your question?

**Comment by bhoskarpooja** [comment]
[![GitHub Streak](https://github-readme-streak-stats.herokuapp.com?user=bhoskarpooja&theme=radical&date_format=%5BY%20%5DM%20j)](https://git.io/streak-stats)
 
  [![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=bhoskarpooja&layout=compact&theme=radical&card_width=445)](https://github.com/anuraghazra/github-readme-stats)
